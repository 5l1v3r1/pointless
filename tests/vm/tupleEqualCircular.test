
output = [a] |> println

a = (123, a)