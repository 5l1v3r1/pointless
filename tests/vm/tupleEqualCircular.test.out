-------------------------------------------------------------------------------
Runtime Error:

Call stack overflow
-------------------------------------------------------------------------------

At (line 4) (column 11) in 'tests/vm/tupleEqualCircular.test'
a = (123, a)
          ^
At (line 4) (column 11) in 'tests/vm/tupleEqualCircular.test'
a = (123, a)
          ^
At (line 4) (column 11) in 'tests/vm/tupleEqualCircular.test'
a = (123, a)
          ^
At (line 4) (column 11) in 'tests/vm/tupleEqualCircular.test'
a = (123, a)
          ^
At (line 4) (column 11) in 'tests/vm/tupleEqualCircular.test'
a = (123, a)
          ^
At (line 4) (column 11) in 'tests/vm/tupleEqualCircular.test'
a = (123, a)
          ^
At (line 4) (column 11) in 'tests/vm/tupleEqualCircular.test'
a = (123, a)
          ^
At (line 4) (column 11) in 'tests/vm/tupleEqualCircular.test'
a = (123, a)
          ^
At (line 4) (column 11) in 'tests/vm/tupleEqualCircular.test'
a = (123, a)
          ^
At (line 4) (column 11) in 'tests/vm/tupleEqualCircular.test'
a = (123, a)
          ^
At (line 4) (column 11) in 'tests/vm/tupleEqualCircular.test'
a = (123, a)
          ^
At (line 4) (column 11) in 'tests/vm/tupleEqualCircular.test'
a = (123, a)
          ^
At (line 4) (column 11) in 'tests/vm/tupleEqualCircular.test'
a = (123, a)
          ^
At (line 4) (column 11) in 'tests/vm/tupleEqualCircular.test'
a = (123, a)
          ^
At (line 4) (column 11) in 'tests/vm/tupleEqualCircular.test'
a = (123, a)
          ^
At (line 4) (column 11) in 'tests/vm/tupleEqualCircular.test'
a = (123, a)
          ^
At (line 4) (column 11) in 'tests/vm/tupleEqualCircular.test'
a = (123, a)
          ^
At (line 4) (column 11) in 'tests/vm/tupleEqualCircular.test'
a = (123, a)
          ^
At (line 4) (column 11) in 'tests/vm/tupleEqualCircular.test'
a = (123, a)
          ^
